# Requests the stamped feature corresponding to the timestamp
# Different modes for reasoning about timestamps are available

# Gets features with closest time preceeding specified
uint8 CLOSEST_BEFORE=0

# Gets features with closest time following specified
uint8 CLOSEST_AFTER=1

# Gets features with closest time to specified
uint8 CLOSEST_EITHER=2

uint8 time_mode
time query_time

---

argus_msgs/FloatVectorStamped features